docs/contributing/translations.md

```diff
@@ -66,6 +66,19 @@ _("Page {page_title} with status {status}".format(page_title=page.title, status=
 
 # Do not use f-string
 _(f"Page {page.title} with status {page.status_string}")
+
+# Do not use {%translate%} or {%blocktranslate%}
```

Comment:
This is probably not as clear as it could be, and this should probably be in its own sub-heading section.

Additionally, you have the code example in a `python` block, but the template tags are used in `html` blocks.

````md

### Passing translated strings into template tag variables

Instead of setting a variable that is the translated string and passing the variable into the template tag as below;

```html
{% comment %} Avoid doing this {% endcomment %}
{% load i18n wagtailadmin_tags %}
{% pageurl page as page_url %}
{% trans 'Visit the live page' as status_title %}
{% status page.status_string url=page_url title=status_title classname="primary" %}
```

it is better to leverage Django's documented approach to passing [translated strings to template tags and filters](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#string-literals-passed-to-tags-and-filters)

```html
{% comment %} Instead do this {% endcomment %}
{% load i18n wagtailadmin_tags %}
{% pageurl page as page_url %}
{% status page.status_string url=page_url title=_("Visit the live page") classname="primary" %}
```

This not only makes the template code easier to read, it also avoids risks of polluting template variables for other code.

````

Note: The django docs link will not be suitable, you will need to see how we link to Django docs in other parts of code and follow that.